public/sitemap.xml

```diff
@@ -1,9 +1,61 @@
 <?xml version="1.0" encoding="UTF-8"?>
-  <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-  <url><loc>https://n.exchange</loc><priority>0.5</priority></url>
-  <url><loc>https://n.exchange/#about</loc><priority>0.5</priority></url>
-  <url><loc>https://n.exchange/#compare</loc><priority>0.5</priority></url>
-  <url><loc>https://n.exchange/terms-and-conditions</loc><priority>0.5</priority></url>
-  <url><loc>https://n.exchange/privacy</loc><priority>0.5</priority></url>
-  <url><loc>https://n.exchange/ico</loc><priority>0.5</priority></url>
-</urlset>
\ No newline at end of file
+<?xml-stylesheet type="text/xsl" href="https://n.exchange/sitemap.xsl"?>
```

Comment:
@nathan analyse this map please

Reply:
You shouldn't include anchor links. They refer to the same page and the sitemap specifies how often that content changes, but it's all one page.

Each page has to have its own title and metadata.

The main page should have links, the exchange, and reviews. (period)

There is no reason to prioritize these links as the website is pretty small and it only works with 2nd tier search engine, Google ignores these things. see instructions here: https://support.google.com/webmasters/answer/183668 (Google ignores <priority> and <changefreq> values, so don't bother adding them.) 

hreflang tags are in place as far as I remember so we do not need to add it to the sitemap.

I guess we're missing:
https://n.exchange/en/faqs
I will send an update for this page by the end of the week and if lucky the end of the day. I know the white label page also has some FAQ so I will suggest you shoot them into FAQ as well as Google will use the FAQ page to show snippets on Google search results.

https://n.exchange/en/signup
If more text will be added on the new design

https://n.exchange/en/signin
If more text will be added on the new design

https://n.exchange/en/instant-white-label
This page definitely needs another link for pricing right next to "contact us" and "live preview". It can not be on the bottom.

When I remove 'en' / 'de' / 'ru' it sends me to the correct language so **I guess** https://n.exchange/signin would be the correct URL for the sitemap.

I could run it through an SEO spider offline when it's ready before submitting it to Google.